gulp/tasks/scripts.js

```diff
@@ -33,6 +35,11 @@ const webpackStream = require( 'webpack-stream' );
  * @returns {PassThrough} A source stream.
  */
 function _processScript( localWebpackConfig, src, dest ) {
+  const cmdLineArgs = gulpUtil.env;
```

Comment:
Deprecated, should be replaced with https://www.npmjs.com/package/minimist (see https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5), though I'm unclear how.